configs/uc-davic.nrel-op.json

```diff
@@ -0,0 +1,83 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://ca-ebike-openpath.nrel.gov",
+        "aggregate_call_auth": "user_only"
+    },
+    "intro": {
+        "program_or_study": "program",
```

Comment:
Let's also add the `opcode` element here so that we can eventually remove the backwards compatibility logic. See `cortezebikes` or similar for an example.
https://github.com/e-mission/nrel-openpath-deploy-configs/blob/1eaadf8a6433a6f282dff09ed055c08060b807e8/configs/cortezebikes.nrel-op.json#L9